build.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
This script has as output: "Build succeeded with warnings", which is followed by "Tests failed". 

(As you already mentioned) The output log/terminal is very hard to read. Is this possible to fix? Some warnings are red some are yellow for example.

Reply:
Locally or on travis?

Reply:
If you run from workspace it works fine